tortoisesvn.json

```diff
@@ -0,0 +1,19 @@
+{
+    "_comment": "Maintainers should update the MSI GUID for uninstaller by navigating to registry path for 64-bit installs, i.e. HKEY_LOCAL_MACHINE/SOFTWARE/Microsoft/Windows/CurrentVersion/Uninstall/",
+    "homepage": "https://tortoisesvn.net",
+    "version": "1.9.5",
+    "license": "GPL",
+    "url": "https://sourceforge.net/projects/tortoisesvn/files/1.9.5/Application/TortoiseSVN-1.9.5.27581-x64-svn-1.9.5.msi",
+    "hash": "dbb3e538fde2fac56781b4893ae6ab45b6cb331d2a3888ed4308a5ec47e85468",
+    "installer": {
+        "args": "/qr"
+    },
+    "post_install": "iex \"cp /Windows/System32/msiexec.exe $dir\"",
```

Comment:
Thanks for the feedback guys! First time contributing to scoop, so still trying to read all the documentation and get familiarized ;)

@lukesampson really appreciate the explanation. Ya, if it were not for the Explorer integration, admin escalation wouldn't be necessary... Will make the changes you suggest!

@deevus would that still be a concern if we install using `TARGETDIR=$dir INSTALLDIR=$dir` ?